src/exports.cc

```diff
@@ -885,6 +885,13 @@ void wasi_unstable_proc_exit(Word /*exit_code*/) {
   context->error("wasi_unstable proc_exit");
 }
 
+Word wasi_unstable_sched_yield() { return 0; }
+
+Word wasi_unstable_poll_oneoff(Word /*in*/, Word /*out*/, Word /*nsubscriptions*/,
+                               Word /*nevents*/) {
+  return 0;
```

Comment:
nit: Could you add a comment saying that keeping nevents at 0 means nothing happened. Is it necessary to set events to 0 explicitly?